### PR TITLE
fix/hide condition for details action buttons

### PR DESCRIPTION
**Pull request by mugnivenko**
https://app.clickup.com/t/1phm2qm hide buttons if status is ended or cancelled
https://app.clickup.com/t/1q2xk7k fix links and buttons for approval page
https://app.clickup.com/t/1q2xpgx redirect route and links for (approval, details) page

### CODE REVIEW

**Comment by mugnivenko** [comment]
https://app.clickup.com/t/1phm2qm

**Comment by mugnivenko** [comment]
https://app.clickup.com/t/1q2xk7k

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
This breaks changes I added. Go to this url on your branch and then on develop: http://localhost:3000/care-package/824186e9-4d72-490e-81ea-39780cb880c3/details 
This package has been ended so you shouldn't be able to see the cancel, end, edit buttons. On develop this is fine, on your branch I can see them
Screenshot 2021-11-03 at 10 29 37
Why have you removed this logic? This is what has broken it.

**Comment by mugnivenko** [comment]
i change condition for buttons in details page (status < 6)

File: pages/care-package/[guid]/details.js [hunk 1]
```diff
@@ -174,14 +174,13 @@ const PackageDetailsPage = () => {
       className="package-details"
       showEditActions
       openedPopup={openedPopup}
-      buttons={[
+      buttons={data?.status < 6 && [
```
Comment (mugnivenko) [inline pages/care-package/[guid]/details.js:177]: i used your condition

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
edit/remove buttons still appear for cancelled/ended package e.g. care-package/e8c5f10f-c791-4cd3-8eaa-14fd1616ca60/details

**Comment by mugnivenko** [comment]
> edit/remove buttons still appear for cancelled/ended package e.g. care-package/e8c5f10f-c791-4cd3-8eaa-14fd1616ca60/details
> 
> The buttons are also in the wrong position

I add condition for (edit, remove) buttons.
I coordinated the change of position with Burak, it's ok.

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Do we want there to be a 'remove' button if there is nothing there to remove?
Screenshot 2021-11-09 at 11 35 49

**Comment by mugnivenko** [comment]
As Burak said to me, i need to merge two link into one link that says "Edit or Remove" with blue style

File: components/Pages/CarePackages/ReviewPackageDetails/PackageInfo.js [hunk 2]
```diff
@@ -25,7 +25,16 @@ const PackageInfo = ({ fncDetails, headerTitle, items, containerId, careChargeCl
 
   return (
     <Container className="review-package-details__items-container">
-      <h3 id={containerId}>{headerTitle}</h3>
+      <Container className='review-package-details__title' display='flex' alignItems='center'>
+        <h3 id={containerId}>{headerTitle || 'Care Package'}</h3>
+        {goToPackage && (
+          <Container className="review-package-details__items-actions" display="flex">
+            <p onClick={goToPackage} className="link-button">
+              Edit or Remove
```
Comment (mugnivenko) [inline components/Pages/CarePackages/ReviewPackageDetails/PackageInfo.js:33]: image

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
You shouldn't be able to edit an approved package e.g. /care-package/471fb9cc-6573-4f72-a90d-21bb7641ef5a/details
Screenshot 2021-11-10 at 10 02 37

**Comment by mugnivenko** [comment]
> You shouldn't be able to edit an approved package e.g. /care-package/471fb9cc-6573-4f72-a90d-21bb7641ef5a/details Screenshot 2021-11-10 at 10 02 37

hide this button if status === 4 (Approved)

**Review by akeamwoodsND — APPROVED** [review]

**Pull request merged**